### ISSUE TITLE
Can't change the colour of the score with Appearance settings for Low Vision Users

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31759]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

-	Whilst a lot can be done to adjust the main user interface, there are limited options when it comes to the actual score:
- Appearance settings only apply to the program controls. 
- The inverted score setting is limited to white music and text on a black background, with no option to customise either element.
- the colour of the score’s paper can be changed, but it seems the colour of the notation cannot be changed. 
Low Vision users have varied needs as to what colours work best for them, a popular choice is often yellow or light orange on a black background as white on black may be to glarrey, but ideally there need to be a range of customisable options so individuals can choose what works for them.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.5.2

### Regression

I was unable to check

### Operating system

iOS: Sequoia 15.6

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
